samples/sampler/npm-shrinkwrap.json

```diff
@@ -14978,6 +14978,16 @@
       "integrity": "sha512-jDctJ/IVQbZoJykoeHbhXpOlNBqGNcwXJKJog42E5HDPUwQTSdjCHdihjj0DlnheQ7blbT6dHOafNAiS8ooQKA==",
       "dev": true
     },
+    "bindings": {
```

Comment:
Don't we only edit this file when we release a version?

Reply:
Good point! It looks updated when I tested this PR. I will drop this change.